.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: java
+
+jdk: 
+- oraclejdk8
+
+before_install:
```

Comment:
Can we move all these instructions to a shell script called `before_install.sh` that lives with this project?

Reply:
That is a good idea! I will make sure I will do that!